lib/msf/base/serializer/readable_text.rb

```diff
@@ -408,7 +460,7 @@ def self.dump_options(mod, indent = '', missing = false)
       next if (opt.evasion?)
       next if (missing && opt.valid?(val))
 
-      tbl << [ name, opt.display_value(val), opt.required? ? "yes" : "no", opt.desc ]
+      tbl << [ name, opt.display_value(val), opt.required? ? format_text("yes", :green) : format_text("no", :red), opt.desc ]
```

Comment:
I'm not sure that coloring these is useful here, but assuming it is, this line is already complicated enough. I'd rather see the formatted thing saved to a local var, e.g.:
```ruby
required_text = opt.required? ? format_text("yes", :green) : format_text("no", :red)
tbl << [ name, opt.display_value(val), required_text, opt.desc ]